test/css/value/unicode.0.json

```diff
@@ -0,0 +1,92 @@
+{
+  "type": "value",
+  "content": [
+    {
+      "type": "ident",
+      "content": "U",
```

Comment:
I think that `checkUnit` is cool. Maybe unicode ranges should get their own node type, what do you think?